docs/content/preview/releases/yba-releases/_index.md

```diff
@@ -55,16 +55,18 @@ For information on YugabyteDB release support timelines, refer to [YugabyteDB re
 
 Keep YBA up-to-date with the latest stable version to get the latest fixes and improvements, as well as to be able to deploy the latest releases of YugabyteDB.
 
-Note that you cannot deploy or upgrade a universe to a version of YugabyteDB that is later than the version of YugabyteDB Anywhere. To upgrade a universe to a more recent version of YugabyteDB, you may first have to upgrade YugabyteDB Anywhere.
+Note that you cannot deploy or upgrade a universe to a higher version of YugabyteDB than the version of YugabyteDB Anywhere. To upgrade a universe to a more recent version of YugabyteDB, you may first have to upgrade YugabyteDB Anywhere.
```

Comment:
You can't use YBA to deploy versions of YugabyteDB that are newer than your YBA instance. To upgrade a universe to a more recent version of YugabyteDB, you may first have to upgrade YBA.